### PR TITLE
Wiz.clean up codes

### PR DESCRIPTION
**Pull request by DreamerWiz**
Fix according to suggestions.
1. Remove `left == reward` logic in allocator.lua.
2. Add config module
3. Add restriction for Mint.Backup
4. Add amount checking for T0_allocation in initializer. Mint for them.
5. Clear Balance-Mirror
6. Change the burn design, transfer to DEAD address, not decrease totalsupply.
7. Fix deploy guides.
8. Process MintedSupply in main.lua

### CODE REVIEW

File: apus_token/main.lua [hunk 1]
```diff
@@ -22,14 +22,18 @@ Token = require('token')
...
+require('config')
 
 -- Function to verify if a message is a mint report from AO Mint Process
 local function isMintReportFromAOMint(msg)
   return msg.Action == "Report.Mint" and msg.From == AO_MINT_PROCESS
 end
 
+-- Function to verify if a message can be processed with the target handler
+local function isMintBackupAuthorized(msg)
+  return msg.Action == "Mint.Backup" and (msg.From == ao.env.Process.Owner or msg.From == ao.id)
```
Comment (jax-cn) [inline apus_token/main.lua:34]: ao.Owner

File: apus_token/mint.lua [hunk 2]
```diff
@@ -117,32 +125,48 @@ Mint.mint = function(msg)
...
-            end, "0", Utils.values(Balances))
+        if not deposits or #deposits == 0 and TotalMintIsZero then
+            -- reset the flag
+            TotalMintIsZero = false
+            -- total mint is zero, which results in no eligible users. Actually the process received mint reports.
+            for i = 1, times do
+                -- Determine the amount to be minted in this iteration
+                local releaseAmount = Mint.currentMintAmount()
+                -- burn the token by adding amount for balance of DEAD address
+                Balances["DEAD"] = BintUtils.add(Balances["DEAD"] or "0", releaseAmount)
```
Comment (jax-cn) [inline apus_token/mint.lua:141]: Might have to be unified with Token.burn
which means either Token.burn be transfer to DEAD Address.
or call Token.burn here.

Comment (DreamerWiz) [reply apus_token/mint.lua:141]: Token.burn is a handler function, should we make a message to call the function?
Changed the implementation in Token.burn, too.

Comment (DreamerWiz) [reply apus_token/mint.lua:141]: > Might have to be unified with Token.burn which means either Token.burn be transfer to DEAD Address. or call Token.burn here.

Token.burn is a handler function, should we make a message to call the function?
Changed the implementation in Token.burn, too.

File: apus_token/main.lua (hunk 1 above)
Comment (DreamerWiz) [reply apus_token/main.lua:34]: > ao.Owner

I have tried but no response in aos console.

**Pull request closed without merging**